<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="DueSensorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Post Angle (Degrees)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;(range 0 - 340)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="DueSensorlcdNumber">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="DueSensorLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Horizontal Pos (mm)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;(range 0 - 500)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="DueSensorlcdNumber_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="DueSensorlcdNumber_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="DueSensorLabel_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Vertical Pos (mm)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;(range 0 - 500)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="DueSensorlcdNumber_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="DueSensorLabel_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Omni Angle (mm)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;(range 0 - 340)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
